app/views/content/terms_candidate.html.erb

```diff
@@ -76,13 +87,7 @@
       <li>Equality and diversity questions</li>
     </ul>
 
-    <p class="govuk-body">You can contact us at becomingateacher@digital.education.gov.uk if you need to update information about:</p>
-
-    <ul class="govuk-list govuk-list--bullet">
-      <li>Qualifications</li>
-      <li>Work experience</li>
-      <li>Safeguarding</li>
-    </ul>
+    <p class="govuk-body">If you make changes to your application after submitting it, you should tell the training provider you applied to.</p>
```

Comment:
`Should tell` sounds a bit casual to me. What about `inform`?

Reply:
"tell" is more plain English than '"inform" and I'd be more concerned with 100% understanding on this page (which no one reads).

Reply:
Ok, plain English it is then.